src/components/Dialog.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <teleport to="body">
+    <div
```

Comment:
The semantic HTML for dialog boxes is `<dialog>` 

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/dialog

```suggestion
    <dialog
```

Reply:
Completed